portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
```

Comment:
Please change here and other places to 2020 (I think it's possible that the templates were also out of date)